src/d_main.c

```diff
@@ -221,6 +221,12 @@ void D_Display (void)
   static int borderdrawcount;
   int wipestart;
   boolean done, wipe, redrawsbar;
+  extern boolean setup_active;
+  extern int menu_background;
+
+  if (!((automapactive && automapoverlay == 2)
```

Comment:
I'd like to get away without this initialization, but I'll try it a solution myself first and the post this here.